tests/integration/.patch_ioredis.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sed -i 's/expect(args\[0\]).to.eql("get")/expect(args\[0\]).to.match(\/get\/i)/' $1
```

Comment:
can you please add a comment expressing the intent of this change?

Reply:
add more comments on all outstanding operations in the scripts and the docker file. This would be removed once I completed [io redis issue](https://github.com/luin/ioredis/issues/1671)